.circleci/config.yml

```diff
@@ -372,6 +393,11 @@ workflows:
           name: test-2.6
           requires:
             - build-2.6
+      - orb/lint:
+          <<: *config-2_6
+          name: lint
+          requires:
+            - build-2.6
```

Comment:
Only concern here is the linting rules for 2.6 might be different than other versions (e.g. new syntax is introduced that is preferred to older versions.) This concern generally applies moving forward to all versions of Ruby, as the same thing could happen in any future version.

Not a huge deal to me, just a potential gap I see.

Reply:
Setting `TargetRubyVersion` to `2.1`, like we do, prevents Rubocop from considering any rules created for versions greater than `2.1`.

Effectively we "locked" Rubocop in the `2.1` era for cops.
Also, we are locked to a specific version of Rubocop, `0.49.1`, so that gem's code is always consistent for us.